Ontologies.Mappings/src/Mappings/v1/Mapped/willow_v1_dtdlv2_mapped.json

```diff
@@ -4586,7 +4586,7 @@
     },
     {
       "InputDtmi": "dtmi:com:willowinc:airport:AirportTerminal;1",
-      "OutputDtmi": "dtmi:mapped:core:Building;1"
+      "OutputDtmi": "dtmi:org:brickschema:schema:Brick:Building;1"
```

Comment:
Why don't we use mapped:Airport here? @connorjcantrell

Reply:
@jbkoh We define Airport as a Site. We previously discussed how Airport Terminal is a building or sub-building. What are your thoughts on mapping AirportTerminal to rec:SubBuilding?

Reply:
Ah sorry, I somehow mixed them up in my head and thought we have terminal on our side. I think we should add it soon. Let's discuss it separately.